src/cli/lib/execute-command.ts

```diff
@@ -1,7 +1,8 @@
-import ora from "ora"
+import { inspect } from "node-inspect-extracted"
```

Comment:
Webpack 4's default polyfill for inspect doesn't support `compact: false`. Not sure what the story is with Webpack 5 & Next.js since default polyfills don't exist in 5.